### PR TITLE
Upgrade Antlr. Tidy up dependencies

### PR DESCRIPTION
**Pull request by aaronbloom**

### CODE REVIEW

File: src/.editorconfig [hunk 1]
```diff
@@ -1,11 +1,10 @@
 # Remove the line below if you want to inherit .editorconfig settings from higher directories
 root = true
 
-# Visual Basic files
```
Comment (aaronbloom) [inline src/.editorconfig:4]: Tidy up the ordering of this

File: src/VCEL.Core/VCEL.Core.csproj [hunk 2]
```diff
@@ -1,32 +1,28 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Authors>Brian Flynn</Authors>
-    <Product>View Client Expression Language</Product>
-    <Description>Spring Expression Like Expression Language</Description>
-  </PropertyGroup>
```
Comment (aaronbloom) [inline src/VCEL.Core/VCEL.Core.csproj:7]: This is duplicate of `src/Directory.Build.props`, which applies to all projects.

File: src/Directory.Packages.props [hunk 3]
```diff
@@ -3,20 +3,18 @@
...
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
     <PackageVersion Include="NUnit" Version="3.14.0" />
-    <PackageVersion Include="NUnit3TestAdapter" Version="4.5.0" />
+    <PackageVersion Include="NUnit3TestAdapter" Version="4.6.0" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageVersion Include="Spectre.Console" Version="0.49.1" />
     <PackageVersion Include="Spring.Core" Version="3.0.2" />
     <PackageVersion Include="System.CodeDom" Version="8.0.0" />
-    <PackageVersion Include="System.Collections.Immutable" Version="8.0.0" />
-    <PackageVersion Include="System.Configuration.ConfigurationManager" Version="8.0.0" />
```
Comment (aaronbloom) [inline src/Directory.Packages.props:20]: Not needed

File: src/Tests/Spel.Benchmark/Spel.Benchmark.csproj [hunk 4]
```diff
@@ -2,14 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```
Comment (aaronbloom) [inline src/Tests/Spel.Benchmark/Spel.Benchmark.csproj:5]: Only need to support net8.0+

File: src/Directory.Build.props [hunk 5]
```diff
@@ -29,6 +30,11 @@
     <NoWarn>
       $(NoWarn),
       NU1701, <!-- Our dependency on Spring.Core -->
+      NUnit1001,
+      NUnit2005,
+      NUnit2007,
+      NUnit2047,
+      NUnit2050,
```
Comment (bgever) [inline src/Directory.Build.props:37]: Why were these necessary?

**Review by bgever — APPROVED** [review]

File: src/VCEL.Core/VCEL.Core.csproj [hunk 6]
```diff
@@ -1,32 +1,30 @@
...
+            <JavaExec Condition="'$(OS)' == 'Windows_NT'">$(JAVA_HOME)\bin\java.exe</JavaExec>
+            <JavaExec Condition="'$(OS)' != 'Windows_NT'">/usr/bin/java</JavaExec>
+        </Antlr4>
+        <Antlr4 Include="Lang\VCELParser.g4">
+            <Error>true</Error>
+            <Listener>false</Listener>
+            <Package>VCEL.Core.Lang</Package>
+            <JavaExec Condition="'$(OS)' == 'Windows_NT'">$(JAVA_HOME)\bin\java.exe</JavaExec>
+            <JavaExec Condition="'$(OS)' != 'Windows_NT'">/usr/bin/java</JavaExec>
+        </Antlr4>
```
Comment (aaronbloom) [inline src/VCEL.Core/VCEL.Core.csproj:27]: Works on windows:

image

File: src/Directory.Build.props (hunk 5 above)
Comment (aaronbloom) [reply src/Directory.Build.props]: Removed
